python/lsst/sdm/schemas/cdb_latiss.yaml

```diff
@@ -307,6 +311,9 @@ tables:
     ivoa:ucd: pos.outline;obs.field
 - name: exposure_flexdata
   "@id": "#exposure_flexdata"
+  description: >
+    The exposure_flexdata table records flexible key/value metadata about each raw image taken.
```

Comment:
Preferred: "Flexible key/value metadata about each raw image taken; used for development and engineering purposes"